datarights.html

```diff
@@ -0,0 +1,122 @@
+as data derived from the process of comparing forecast fields to
+observations (e.g. summary statistics).
+
+
+Guiding Principles
+------------------
+
+The following principles guide our data policies.
+
+1. Organizations retain ownership of the data they upload to the framework.
+2. Users upload data to the framework on behalf of organizations.
+2. Users have complete control over how their data may be accessed by other users.
+2. Users may delete data from the framework.
+2. A forecast trial may require that data owners relinquish control of a
+   copy of their data (optionally anonymized) to ensure trial fairness,
+   transparency, and reproducibility. The terms of a trial and its data
+   permissions will be fully defined before users contribute data to it.
+2. The framework will not sell the data that it controls.
+2. All non-public data will be securely deleted by the conclusion of the DOE
+   funding period (June 30, 2021).
+
+
+Contributing data
+-----------------
+
+Each organization contributing data to the framework retains ownership
+and control of its data. Each framework user is a member of an
+organization and contributes data on behalf of the organization.
+
+Through the framework web interface or API calls, the user associates
+data with data access roles. These roles allow specific organizations/users
+to access to the data. Example roles are shown below. The web interface
+will show an organization a list of all of the submitted data/metadata
+it owns and the roles that can access each data source.
+
+
+Deleting data
+-------------
+
+An organization may delete its data from the framework at any time, or
+an organization may ask the framework administrators to delete its data
+from the framework.
+
+Anonymized time series data and summary statistics
+derived from the data are owned by the framework and not deleted.
+(Framework is not allowed to sell anonymized data or statistics.)
+
+All non-public data will be securely deleted at the conclusion of the
+DOE funding period (June 30, 2021). The post-DOE-funding framework
+administrators and data contributors will need to resign new NDAs.
+
+Users may delete data sooner than the end of the funding period.
+
+
+Data access roles
+-----------------
+
+A data access role defines the ways in which the data can accessed by users.
+The framework supports the following data access roles, ordered from least
+to most permissible.
+
+* No access
+    * Default access level for data uploaded by users.
+* Peer-to-peer
+    * A single user grants another user permission to view data.
+* Multiparty Confidential Access anonymized
+    * Groups of organizations/users will be defined in response to each need
+      e.g. a particular forecast trial.
+    * Anonymized data accessible is to all users within a specific group
+    * Anonymized data not accessible by users outside of the group (or general public)
+    * True identities not known by framework administrators (see note below)
```

Comment:
Make this more clear. I thought it was the ownership of anonymized data that no one knows through the use of anonymized role to upload